Products/ZenModel/skins/zenmodel/editUserSettings.pt

```diff
@@ -322,24 +287,12 @@
                         </td>
```

Comment:
Should we just remove the change-password elements from the ui for CZ?

Reply:
That's intentional, until we fully divest ourselves of local users. For the moment, that's how we have to develop.

Reply:
But only CZ_ADMIN and up have it, so users wouldn't see it even if it did go out.